lamorel/src/lamorel/server/llms/hf_llm.py

```diff
@@ -250,6 +250,12 @@ def forward(self, module_function_keys, contexts, candidates=None, require_grad=
         _forward_results = [[] for _ in range(len(contexts))]
         if candidates is None:
             candidates = [[""] for _ in range(len(contexts))]
+        elif len(candidates) == 0 and len(contexts) > 0:
+            candidates = [[""] for _ in contexts]
```

Comment:
Why did you add this `elif` clause? Isn't the `else` enough?

Reply:
Well not exactly. The `elif` authorizes `candidates=[]`. I guess I could merge it with the `if` above with an `and` operator.